oauth2/oauth2-jose/src/main/java/org/springframework/security/oauth2/jwt/JwtAudienceValidator.java

```diff
@@ -32,19 +32,30 @@ public final class JwtAudienceValidator implements OAuth2TokenValidator<Jwt> {
 
 	private final JwtClaimValidator<Collection<String>> validator;
 
+	private boolean allowEmpty;
```

Comment:
Having thought about this further, I think we should wait on adding this toggle to validators that require the claim by default. We can relax it when a clear need arises.